### DISCUSSION TITLE
Converting a column to numeric

### DISCUSSION THREAD

**Discussion by bboreham** [grafana/grafana#37147] [Dashboards, Panels & Visualizations]
I am starting with logs in Loki, and trying to show a table of slowest queries.
Then I want to sort by duration, to show the slowest ones at the top.

However, after applying a "Labels to fields" Transformation, I seem to get all string-type columns, which means sorting is alphabetic.  How do I get one field to be numeric?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
